### PR TITLE
[Test] : pytorch_vision_meal_v2.md 번역

### PR DESCRIPTION
**Pull request by woodywarhol9**
## 라이선스 동의
_변경해주시는 내용에 BSD 3항 라이선스가 적용됨을 동의해주셔야 합니다._<br />
_더 자세한 내용은 [기여하기 문서](https://github.com/PyTorchKorea/hub-kr/blob/master/CONTRIBUTING.md)를 참고해주세요._<br />
_동의하시면 아래 `[ ]`를 `[x]`로 만들어주세요._<br />

- [x] 기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.

## 관련 이슈 번호
_이 Pull Request와 관련있는 이슈 번호를 적어주세요._<br />
_이슈 또는 PR 번호 앞에 #을 붙이시면 제목을 바로 확인하실 수 있습니다. (예. #999 )_

- **이슈 번호**: #(숫자)

## PR 종류
_이 PR에 해당되는 종류 앞의 `[ ]`을 `[x]`로 변경해주세요._<br />
- [ ] 오탈자를 수정하거나 번역을 개선하는 기여
- [X] 번역되지 않은 모델 소개를 번역하는 기여
- [ ] 공식 허브의 내용을 반영하는 기여
- [ ] 위 종류에 포함되지 않는 기여

## PR 설명
2주차 개인 과제 : MEAL-V2 번역

### CODE REVIEW

File: pytorch_vision_meal_v2.md [hunk 1]
```diff
@@ -17,37 +17,34 @@ order: 10
 demo-model-link: https://huggingface.co/spaces/pytorch/MEAL-V2
 ---
 
-We require one additional Python dependency
+추가로 1개의 파이썬 패키지를 설치해야 합니다.
```
Comment (cpprhtn) [inline pytorch_vision_meal_v2.md]: 해당글을 처음 들어왔는데 바로 `추가로`라는 단어로 시작하는게 뭔가 글 중간부터 읽는 느낌입니다!

하나의 추가적인 패키지가 필요합니다. or 하나의 추가적인 패키지가 요구됩니다. 등의 느낌으로 수정하면 좋을 것 같아요.!!

File: pytorch_vision_meal_v2.md [hunk 2]
```diff
@@ -17,37 +17,34 @@ order: 10
...
 ```bash
 !pip install timm
 ```
 
 ```python
 import torch
-# list of models: 'mealv1_resnest50', 'mealv2_resnest50', 'mealv2_resnest50_cutmix', 'mealv2_resnest50_380x380', 'mealv2_mobilenetv3_small_075', 'mealv2_mobilenetv3_small_100', 'mealv2_mobilenet_v3_large_100', 'mealv2_efficientnet_b0'
-# load pretrained models, using "mealv2_resnest50_cutmix" as an example
+# 모델 종류: 'mealv1_resnest50', 'mealv2_resnest50', 'mealv2_resnest50_cutmix', 'mealv2_resnest50_380x380', 'mealv2_mobilenetv3_small_075', 'mealv2_mobilenetv3_small_100', 'mealv2_mobilenet_v3_large_100', 'mealv2_efficientnet_b0'
+# 사전에 학습된 "mealv2_resnest50_cutmix"을 로딩하는 예시입니다.
```
Comment (cpprhtn) [inline pytorch_vision_meal_v2.md]: `로딩하는` 보다는 `불러오는`이라는 단어가 번역상에는 더 예쁠 것 같습니다!

File: pytorch_vision_meal_v2.md (hunk 1 above)
Comment (cpprhtn) [reply pytorch_vision_meal_v2.md]: 아래 `# sample execution (requires torchvision)` 문장에서 필요 라고 번역하셨으므로 하나의 추가적인 패키지가 필요합니다. 쪽이 더 나은거 같습니다.

File: pytorch_vision_meal_v2.md [hunk 3]
```diff
@@ -58,32 +55,32 @@ preprocess = transforms.Compose([
...
-# move the input and model to GPU for speed if available
+# 가능하다면 속도를 위해 입력과 모델을 GPU로 옮깁니다.
 if torch.cuda.is_available():
     input_batch = input_batch.to('cuda')
     model.to('cuda')
 
 with torch.no_grad():
     output = model(input_batch)
-# Tensor of shape 1000, with confidence scores over Imagenet's 1000 classes
+# shape이 1000이며 ImageNet의 1000개 클래스에 대한 신뢰도 점수(confidence score)가 있는 Tensor
```
Comment (cpprhtn) [inline pytorch_vision_meal_v2.md]: `1000개의  ImageNet 클래스에 대한 신뢰도 점수(confidence score)를 가진 1000 크기의 Tensor`

라고 저는 번역할듯한데,,, 해당문장은 좀 매끄럽게 번역하기 어렵네요.ㅠㅠ

File: pytorch_vision_meal_v2.md [hunk 4]
```diff
@@ -58,32 +55,32 @@ preprocess = transforms.Compose([
...
 print(output[0])
-# The output has unnormalized scores. To get probabilities, you can run a softmax on it.
+# output엔 정규화되지 않은 신뢰도 점수가 있습니다. 확률 값을 얻으려면 소프트맥스를 실행하세요.
```
Comment (cpprhtn) [inline pytorch_vision_meal_v2.md]: 팀별과제: 번역 규칙 관련 제안점 모으기 (용어집 추가하기)에 의견 내볼만한 글이긴한데.
`Activation function`이나 `optimizer`관련 용어는 번역 안하는게 더 이해하기 좋을것 같다는 개인생각 한번 얘기해봅니당.

> 확률 값을 얻으려면 softmax를 실행하세요.

**Review by cpprhtn — CHANGES_REQUESTED** [review]
한번 읽어보시고 같이 고민해봅시다!!!
제 개인적인 의견이니.. 꼭 반영해볼 필요없습니당

File: pytorch_vision_meal_v2.md (hunk 1 above)
Comment (Lee-JaeWon) [inline pytorch_vision_meal_v2.md]: 패키지가 **종속성을 가진다는 것**을 번역에서 내포해도 좋을 것 같은데, 이런 부분도 함께 이야기해보면 좋을 것 같습니다!

Comment (cpprhtn) [reply pytorch_vision_meal_v2.md]: 네네 저도 패키지라고 번역해두셔서 물어보려 하긴했네요 😆

Comment (woodywarhol9) [reply pytorch_vision_meal_v2.md]: 다시 읽어보니 확실히 뜬금없긴 하네요..!

Comment (woodywarhol9) [reply pytorch_vision_meal_v2.md]: 넵 종속성에 대해서 다루는 것이 더 좋겠네요~
